docs/tutorial/part-one/index.md

```diff
@@ -70,9 +70,11 @@ This is huge because it makes development so much faster and fun.
 
 Let's try it.
 
-Try changing "Hello world!" in the page component to "Hello Gatsby!". You'll find the page component at `src/pages/index.js`. The text in your browser should change within a second.
+You'll need code editing software for the next part of this tutorial. <a href="https://code.visualstudio.com/">VS Studio</a>  is a nice one. After downloading and opening a code editing software, open a folder on your computer called "tutorial-part-one." This folder was automatically created when you ran the previous terminal commands in the first part of this tutorial.
 
-Try some other tricks.
+Now that you've opened the "tutorial-part-one" folder in your code editing software, it's time to edit your website. In the code editing software, you'll need to find the 'src/pages/index.js' page. Once you find that, try changing "Hello world!" in the page component to "Hello Gatsby!". The text in your browser should change within a second.
```

Comment:
how about "to find the file at `src/pages/index.js`"

Reply:
In markdown, you can make text appear as inline code by surrounding it with backticks. I'd do that around "src/pages/index.js".